src/helpers/TextFormatting.js

```diff
@@ -143,33 +143,38 @@ export function linkifyChannels(word) {
 
 export function linkifyUsers(word, userlist) {
     let ret = '';
-    let nick = '';
+    let user = '';
     let prepend = '';
     let append = '';
     let punc = ',.!:;-+)]?¿\\/<>@';
     let validLastChar = punc.indexOf(word[word.length - 1]) > -1;
     let normWord = word.toLowerCase();
     let hasProp = Object.prototype.hasOwnProperty;
 
-    // Checking for a nick in order of processing cost
+    // Checking for a user in order of processing cost
     if (hasProp.call(userlist, normWord)) {
-        nick = word;
+        user = userlist[normWord];
     } else if (hasProp.call(userlist, normWord.substr(0, normWord.length - 1)) && validLastChar) {
         // The last character is usually punctuation of some kind
-        nick = word.substr(0, word.length - 1);
+        user = userlist[normWord.substr(0, normWord.length - 1)];
         append = word[word.length - 1];
     } else if (hasProp.call(userlist, _.trim(normWord, punc))) {
-        nick = _.trim(word, punc);
-        let nickIdx = word.indexOf(nick);
-        append = word.substr(nickIdx + nick.length);
+        user = userlist[_.trim(normWord, punc)];
+        let nickIdx = normWord.indexOf(user.nick.toLowerCase());
+        append = word.substr(nickIdx + user.nick.length);
         prepend = word.substr(0, nickIdx);
     } else {
         return word;
     }
 
-    let escaped = _.escape(nick);
-    let colour = createNickColour(nick);
-    ret = `<a class="kiwi-nick" data-nick="${escaped}" style="color:${colour}">${escaped}</a>`;
+    let escaped = _.escape(user.nick);
+    let colour = user.colour;
+
+    ret = `<a class="kiwi-nick" data-nick="${escaped}"`;
+    if (colour) {
+        ret += `style="color:${colour}"`;
```

Comment:
This *shouldn't* be an issue but it smells a bit funny. There is no space between the HTML attributes when put together. Adding a space to the end of the string on line 173 and 175 will solve this.